aten/src/ATen/templates/Tensor.h

```diff
@@ -165,8 +166,8 @@ class CAFFE2_API Tensor {
   int64_t ndimension() const {
     return dim();
   }
-  bool is_contiguous() const {
-    return impl_->is_contiguous();
+  bool is_contiguous(at::MemoryFormat memory_format=at::MemoryFormat::Any) const {
```

Comment:
why default value here is Any and not Contiguous? doesn't really matter, but might help readability

Reply:
See my comment above -- I think @VitalyFedyunin is going to look into removing `Any`.